broker-plugins/jdbc-store/src/main/java/org/apache/qpid/server/store/jdbc/JDBCDetails.java

```diff
@@ -379,7 +403,8 @@ public boolean isOverridden()
                 return contextMap.containsKey(CONTEXT_JDBCSTORE_USEBYTESFORBLOB)
                         || contextMap.containsKey(CONTEXT_JDBCSTORE_BIGINTTYPE)
                         || contextMap.containsKey(CONTEXT_JDBCSTORE_VARBINARYTYPE)
-                        || contextMap.containsKey(CONTEXT_JDBCSTORE_BLOBTYPE);
+                        || contextMap.containsKey(CONTEXT_JDBCSTORE_BLOBTYPE)
```

Comment:
Please undo the change as context variable for timestamp type should not be used to indicate to use bytes methods for blob.

Reply:
The method modified here is isOverridden() not isUseBytesMethodForBlob()